db/migrate/20191001120743_add_fields_to_reported_data_facet.rb

```diff
@@ -0,0 +1,15 @@
+class AddFieldsToReportedDataFacet < ActiveRecord::Migration[5.2]
+  def up
+    add_column :host_facets_reported_data_facets, :virtual, :boolean
+    add_column :host_facets_reported_data_facets, :sockets, :integer
+    add_column :host_facets_reported_data_facets, :cores, :integer
+    add_column :host_facets_reported_data_facets, :ram, :float
```

Comment:
I'm wondering if we should store the number of bytes in an integer. I think we have helpers to humanize bytes.